### PR TITLE
[FRONT-74] Adapt recharts design to our theme & implement PR feedback & implement design changes

### PR DESCRIPTION
**Pull request by DamianGrosz2**
### Description of Issue
The placement of the buttons above the chart has been improved to enhance the overall layout. 
The date format has been updated to "Month Year" for improved readability. 
Lastly, the design of the chart has been updated.
Also the design of the details page has been updated

### How I implemented
To achieve these improvements, the following changes were made:

Adjusted the positioning of the buttons to be placed above the chart.
Updated the date format.
Made design updates to enhance the visual aesthetics.

### Other

Please review and merge this pull request.

### CODE REVIEW

**Review by pczern — APPROVED** [review]
it's an improvement to what we have, so direct approval. Work in progress when it comes to number of lines per component, and some redundant type that can be adjusted later though

**Review by cnk1dev — CHANGES_REQUESTED** [review]
Screenshot 2023-05-26 at 18 26 54

Please improve the chart tooltip (if possible):
- reduce font size to match rest of page
- remove "value: "
- add border-radius
- use same border-color as buttons

**Comment by DamianGrosz2** [comment]
> it's an improvement to what we have, so direct approval. Work in progress when it comes to number of lines per component, and some redundant type that can be adjusted later though

Yes but also this depends still on the backend data so when we have that the types will change

File: src/components/page/details/Chart.tsx [hunk 1]
```diff
@@ -139,33 +150,50 @@ const Chart = ({ starData, forkData, issueData }: ChartProps) => {
...
+              left: -15,
+              bottom: 5
+            }}
+          >
+            <CartesianGrid strokeDasharray="3 3" vertical={false} stroke="#2C2D3C" />
+            <XAxis
+              dataKey="name"
+              tick={{ fontSize: '12', fontWeight: 'light' }}
+              tickFormatter={formatDate}
+              stroke="#858699"
```
Comment (cnk1dev) [inline src/components/page/details/Chart.tsx:182]: Please use Tailwind colors instead of hex values

**Review by cnk1dev — CHANGES_REQUESTED** [review]
Screenshot 2023-05-26 at 22 11 13


Better but still luft nach oben 🥸
- Make the value (not the date) in the tooltip bold and add a few pixels of spacing above it
- Make the vertical indicator line which appears on hover grey
- Reduce font size of purple "value" at the bottom

Sorry for being so stingy but Viet is very focused on design so let's try to make it perfect

**Review by cnk1dev — APPROVED** [review]
Still uses hard-coded hex values in Chart.tsx, I'll approve for now and change that later myself though

**Pull request merged**